styles/components/_search.scss

```
@@ -141,6 +141,15 @@
       padding: 0px;
     }
   }
+
+  .o-acp__item {
+    padding: 0;
+    @include ktheme() {
+      background: theme('background-color');
+      color: theme('text-color')!important;
+
+    }
```

```suggestion
      background: unset;
      color: unset;
```
How about this? I guess we do not need to set colour again here.